src/containers/email/index.tsx

```diff
@@ -1,41 +1,101 @@
 import React, { useState } from 'react';
 import { Helmet } from 'react-helmet';
 import styled from 'styled-components';
-import { Select, Typography } from 'antd';
+import { Select, Typography, Row, Col, Button, Spin, Alert } from 'antd';
 import { Routes } from '../../App';
 import PageLayout from '../../components/pageLayout';
 import { ReturnButton } from '../../components/themedComponents';
 import PageHeader from '../../components/pageHeader';
-import { EmailType, EmailerFilters } from './types';
+import {
+  EmailType,
+  EmailerFilters,
+  FilteredSite,
+  FilterSitesParams,
+} from './types';
 import EmailerFilterControls from '../../components/emailerFilterControls';
+import AdoptedSitesTable from '../../components/adoptedSitesTable';
+import protectedApiClient from '../../api/protectedApiClient';
+import { NEIGHBORHOOD_OPTS, Neighborhoods } from '../../assets/content';
 import SendEmailForm from '../../components/forms/sendEmailForm';
-// import protectedApiClient from '../../api/protectedApiClient';
 
 const EmailPageContainer = styled.div`
   width: 90vw;
   margin: 30px auto auto;
 `;
 
-const selectStyles = { marginTop: 20, minWidth: 150 };
+const FilterHeader = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin-top: 15px;
+`;
+
+const FetchInfoContainer = styled.div`
+  text-align: center;
+  padding: 30px;
+`;
+
+const selectStyles = { marginTop: 20, minWidth: 150, marginBottom: 20 };
```

Comment:
❓Just curious, what's the reason for doing this instead of using styled components?

Reply:
For some reason, when I used a styled component, typescript would keep raising errors with the provided `onChange` function - iirc it's because `Select` has a parameterized value type, and as styled component that value type became unknown so it'd raise an error.

Reply:
Can you see if this works?
```
import { SelectProps } from 'antd/lib/select';

...

const EmailTypeSelect = styled((props: SelectProps) => <Select {...props} />)`
  margin-top: 20px;
  min-width: 150px;
  margin-bottom: 20px;
`;
```

and using `<EmailTypeSelect ... />`?